packages/client/src/trackCCTX.ts

```diff
@@ -58,15 +58,16 @@ const fetchCCTXData = async function (
   json: Boolean
 ) {
   const cctx = await getCCTX(hash, API);
-  const receiver_chainId = cctx?.outbound_tx_params[0]?.receiver_chainId;
-  const outbound_tx_hash = cctx?.outbound_tx_params[0]?.outbound_tx_hash;
+  const receiver_chainId = cctx?.outbound_params[0]?.receiver_chainId;
+  const outbound_tx_hash = cctx?.outbound_params[0]?.outbound_tx_hash;
   let confirmed_on_destination = false;
+  console.log(cctx?.outbound_params);
```

Comment:
should we remove this log?

Reply:
Yeah, sorry, has been removed https://github.com/zeta-chain/toolkit/pull/170/commits/d5db87bcb3438de5308224b0011093357a8f6cd9